src/card.tsx

```diff
@@ -27,8 +31,9 @@ export const Card = forwardRef<
         'grid-cols-1',
         'gap-4',
         isFlat && 'mx-auto text-center max-w-78',
-        isFlat && 'px-4 py-8',
-        !isFlat && 'p-4 sm:p-6',
+        padding === 'large' && 'px-4 py-8',
```

Comment:
Nit: Should we add these as custom styles and make the className dynamic instead? (wondering if this padding css could be made reusable).